package.json

```diff
@@ -33,17 +33,17 @@
   "homepage": "https://github.com/jalantechnologies/boilerplate-node-ts#readme",
   "dependencies": {
     "@sendgrid/mail": "^7.6.1",
-    "@types/body-parser": "^1.19.2",
-    "@types/mongoose": "^5.11.97",
     "bcrypt": "^5.0.1",
     "body-parser": "^1.19.1",
-    "config": "^3.3.7",
+    "config": "^3.3.6",
     "express": "^4.17.2",
     "google-libphonenumber": "^3.2.27",
     "jsonwebtoken": "^8.5.1",
+    "jws": "^4.0.0",
```

Comment:
Why is this needed?

Reply:
on doing ```npm run test``` I was getting an error to install jws npm package therefore I installed it to remove that error.